Makefile

```diff
@@ -53,7 +53,8 @@ OBJECTS+=gpio.o compatibility.o
 
 else ifeq "$(RPI)" "1"
 DRIVER_DIR=$(ARCH_DIR)/RPi
-OBJECTS+=bcm2835.o	
+OBJECTS+=bcm2835.o
+OBJECTS+=interrupt.o
 # The recommended compiler flags for the Raspberry Pi
```

Comment:
Adding ``` SHARED_LINKER_FLAGS+=-pthread  ``` removes the need to include -pthread in the makefiles for examples etc. in this and additional libraries.